### DISCUSSION TITLE
How you do HEAD request?

### DISCUSSION THREAD

**Discussion by lebzm** [rest-nvim/rest.nvim#496] [Q&A]
I try using HEAD but the response showing that the curl is using GET.

```
HEAD http://localhost:8080
```

**Comment by boltlessengineer** [comment]
Sorry for the late reply, I've been busy for a while.

That's correct syntax and I can confirm that working.
Can you check if you get different response when running this command:
```console
curl -sL -v http://localhost:8080 -X HEAD -H "User-Agent: rest.nvim v3.8.4" -w "%{stderr}? time_total:%{time_total}\n? size_download:%{size_download}\n"
```
This is exact command `rest.nvim` runs internally when you fire that request.
